app/partials/errorDetail.html

```diff
@@ -0,0 +1,20 @@
+<table class="detail-table" ng-if="error">
+    <thead>
+        <tr>
+            <th class="id" scope="col">Loan/Application Number</th>
+            <th class="description" scope="col">Field</th>
+            <th class="value" scope="col">Submitted Value</th>
+            <th class="action" scope="col">Action</th>
+        </tr>
+    </thead>
+    <tbody ng-repeat="item in error.errors">
+        <tr ng-repeat="(key, value) in item.properties">
```

Comment:
This needs to be corrected to display like the wireframes. I'll follow up offline.